app/lib/search_service.dart

```diff
@@ -81,14 +81,27 @@ class SearchService {
         var versionKeys =
             packages.where(exists).map((p) => p.latestVersionKey).toList();
         if (versionKeys.isNotEmpty) {
+          // select latest development versions for each package
+          var packageKeys = versionKeys.map((vk) => vk.parent).toList();
+          List<PackageVersion> devVersions = await Future.wait(
```

Comment:
Strictly speaking, this adds more latency then necessary, because it first does `await db.lookup(keys)`, waits for the result and then does `await Future.wait(...)`.

The second call will download all versions of all `[packages]` and contains therefore also the information from the first rpc.